README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Version npm](https://img.shields.io/npm/v/nodify-shopify.svg)](https://www.npmjs.com/package/nodify-shopify)
 [![Build Status](https://img.shields.io/travis/microapps/Nodify-Shopify/master.svg)](https://travis-ci.org/microapps/Nodify-Shopify)
+[![bitHound Score](https://www.bithound.io/github/microapps/Nodify-Shopify/badges/score.svg)](https://www.bithound.io/github/microapps/Nodify-Shopify/)
+[![Dependency Status](https://david-dm.org/microapps/Nodify-Shopify.svg "Dependencies Checked & Updated Regularly (Security is Important!)")](https://david-dm.org/microapps/Nodify-Shopify)
```

Comment:
Hi @carlosperez, thank you.
I noticed that bitHound isn't available on shields.io, I was talking about David here.
We will definitely use the official badge for bitHound.